### DISCUSSION TITLE
How Can I Access GitHub Education, With No Dated Documents In Hand ?

### DISCUSSION THREAD

**Discussion by Mohammad25Ayman2002** [community/community#139554] [GitHub Education]
### Hi everyone,

Hello, I need Help Accessing GitHub Education, I tried lots of methods to pass the document verification, used photos of my college ID Card, screen shots of my college's website with my name on it, and tried a photo of an official paper that proves that I'm enrolled in that college but it contains no dates. none of these worked for me I don't know what to do, please help

**Comment by davevad93** [comment]
Hi @Mohammad25Ayman2002 ,

An academic document with a current date in a `DD/MM/YYYY` or  `MM/DD/YYYY` format is required to apply, as stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student) "***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***"

You can use any of the above documents with a current date on it, for example, I submitted a screenshot of my last exam results and I was approved, if you can do the same I suggest you to do so.

**Reply by Mohammad25Ayman2002** [reply]
should it be in english or any language is ok ?

**Reply by davevad93** [reply]
Hey @Mohammad25Ayman2002 , it's better to translate the academic documentation to English. Attach a screenshot of the original and the translated one.
